packages/react-components/react-tags/src/components/TagButton/useTagButton.tsx

```diff
@@ -14,47 +13,12 @@ import { Avatar } from '@fluentui/react-avatar';
  * @param ref - reference to root HTMLElement of TagButton
  */
 export const useTagButton_unstable = (props: TagButtonProps, ref: React.Ref<HTMLElement>): TagButtonState => {
-  const {
-    checked = false,
-    disabled = false,
-    dismissable = false,
-    shape = 'rounded',
-    size = 'medium',
-    appearance = 'filled-lighter',
-  } = props;
-
-  return {
-    components: {
-      root: 'div',
-      contentButton: 'button',
-      avatar: Avatar,
-      icon: 'span',
-      primaryText: 'span',
-      secondaryText: 'span',
-      dismissButton: 'button',
+  const state = useTag_unstable(props, ref);
```

Comment:
Couldn't you use do

```ts
useTag_unstable({ tabIndex: 0, ...props }, ref)
```

Reply:
I think it would need to be 
```ts
useTag_unstable({ ...props, content: { tabIndex: 0, ...props.content } }, ref)
``` 
or make `content` the primary slot.

Reply:
Can you remind me what the distinction between `Tag` and `TagButton` is?

If it's just the `tabIndex` maybe we should look at a way to implement this behavior as a prop on `Tag`?

Reply:
I'm having doubts on TagButton as well. 
- TagButton looks and works really similar as a split button: it has two focusable parts, the content and the 'x' icon. While Tag has only the 'x' icon focusable. But this can be done by an extra prop on Tag. 
- there're style differences, TagButton has an extra divider and different focus rings, but this is nothing that can't be solved by style override.
- The deal breaker for me - Tag doesn't have select state but TagButton can. This makes me think TagButton as a separate component can be cleaner. 
When I started working on Tag, there's already a TagButton component so I figure it would be easier to keep it as is. And maybe merge these two components as we see how things goes. It feels easier than creating one component and split it when we must.

Reply:
And also content doesn't like to be spreaded :/ :
<img width="828" alt="image" src="https://user-images.githubusercontent.com/28751745/235578746-1a1f18f6-3195-4185-a622-69598ec613ef.png">

Reply:
Maybe it's just a typing issue? Anythings @behowell ?

Reply:
Yeah this works :) updated